server/src/main/java/org/apache/cassandra/sidecar/cluster/instance/InstanceMetadata.java

```diff
@@ -66,6 +72,26 @@ public interface InstanceMetadata
      */
     @NotNull CassandraAdapterDelegate delegate() throws CassandraUnavailableException;
 
+    /**
+     * @return commitlog directory of the cassandra instance
+     */
+    @Nullable String commitlogDir();
+
+    /**
+     * @return hints directory of the cassandra instance
+     */
+    @Nullable String hintsDir();
+
+    /**
+     * @return saved caches directory of the cassandra instance
+     */
+    @Nullable String savedCachesDir();
+
+    /**
+     * @return local system data file directory of the cassandra instance
+     */
+    @Nullable String localSystemDataFileDir();
```

Comment:
I don't really know what this maps to in the cassandra.yaml configuration. Did I miss something?

Reply:
It maps to `local_system_data_file_directory` in [cassandra.yaml](https://github.com/apache/cassandra/blob/trunk/conf/cassandra.yaml#L407C3-L407C35).